src/repl/session.rs

```diff
@@ -679,6 +680,47 @@ impl Session {
         output.append(&mut result);
     }
 
+    #[cfg(feature = "cli")]
+    fn reload(&mut self, output: &mut Vec<String>) {
+        self.asts.clear();
+        self.contracts.clear();
+        self.costs_reports.clear();
+        self.coverage_reports.clear();
+        self.executed.clear();
```

Comment:
`executed` is just a list of the commands which have been executed. Let's not clear that one.

Reply:
Thank for the info, I wasn't aware of that. 👍